weather_finder/weather_finder.py

```diff
@@ -1,85 +1,85 @@
 import requests
 
-API_KEY = '9104e0a25f676b56ac31901d10c5cf0a'
-
-
-def verify_city(json_response):
-    """
-    Takes in response from OWM and checks if city exists
 
-    Arguments:
-       json_response (json): contains response of a request from OWM
-    """
-
-    if json_response['cod'] == '404':
-        print 'City could not be found, please retry'
-	return False
-    else:
-	print 'City found'
-	return True
+API_KEY = '9104e0a25f676b56ac31901d10c5cf0a'
+location = 'London'
+location_url = ('http://api.openweathermap.org/data/2.5/weather?q={}&APPID={}').format(location, API_KEY)
 
 
 def get_weather_info(location):
     """
-    Gets information about the weather of a location and displays it
+    Gets information about the weather of a location and displays it.
 
     Arguments:
         location (str): a string that specifies which place to search weather
+
     """
-	
-    location_url = 'http://api.openweathermap.org/data/2.5/weather?q=' + location + '&APPID=' + API_KEY	
     weather_raw_data = requests.get(location_url)
     weather_json = weather_raw_data.json()
     if not verify_city(weather_json):
-	return
+        return None
     else:
-	data_output(data_organizer(weather_json))
- 
+        return get_organized_weather_data(weather_json)
+
 
-def data_organizer(weather_raw_data):
+def verify_city(json_response):
     """
-    Gets information about the weather of a location and displays it
+    Takes in response from OWM and checks if city exists.
 
     Arguments:
-        location (str): a string that specifies which place to search weather
+       json_response (json): contains response of a request from OWM
+
     """
+    if json_response['cod'] == '404':
+        print('City could not be found, please retry')
+        return False
+
+    return True
 
+
+def get_organized_weather_data(raw_weather_data):
```

Comment:
Please rename the argument name to just `weather_data` as you are calling this method above like:
```
get_organized_weather_data(weather_json)
```